pkg/providers/imagefamily/bootstrap/parser_helpers.go

```diff
@@ -117,83 +116,56 @@ func getFuncMapForContainerdConfigTemplate() template.FuncMap {
 	}
 }
 
-func getStringFromVMType(enum nbcontractv1.VmType) string {
+func getStringFromVMType(enum nbcontractv1.ClusterConfig_VM) string {
 	switch enum {
-	case nbcontractv1.VmType_VM_TYPE_STANDARD:
+	case nbcontractv1.ClusterConfig_STANDARD:
 		return "standard"
-	case nbcontractv1.VmType_VM_TYPE_VMSS:
+	case nbcontractv1.ClusterConfig_VMSS:
 		return "vmss"
 	default:
 		return ""
 	}
 }
 
-func getStringFromNetworkModeType(enum nbcontractv1.NetworkModeType) string {
+func getStringFromNetworkPluginType(enum nbcontractv1.NetworkPlugin) string {
 	switch enum {
-	case nbcontractv1.NetworkModeType_NETWORK_MODE_TRANSPARENT:
-		return "transparent"
-	case nbcontractv1.NetworkModeType_NETWORK_MODE_BRIDGE:
-		return "bridge"
-	default:
-		return ""
-	}
-}
-
-func getStringFromNetworkPluginType(enum nbcontractv1.NetworkPluginType) string {
-	switch enum {
-	case nbcontractv1.NetworkPluginType_NETWORK_PLUGIN_TYPE_AZURE:
+	case nbcontractv1.NetworkPlugin_NP_AZURE:
 		return "azure"
-	case nbcontractv1.NetworkPluginType_NETWORK_PLUGIN_TYPE_KUBENET:
+	case nbcontractv1.NetworkPlugin_NP_KUBENET:
 		return "kubenet"
 	default:
 		return ""
 	}
 }
 
-func getStringFromNetworkPolicyType(enum nbcontractv1.NetworkPolicyType) string {
+func getStringFromNetworkPolicyType(enum nbcontractv1.NetworkPolicy) string {
 	switch enum {
-	case nbcontractv1.NetworkPolicyType_NETWORK_POLICY_TYPE_AZURE:
+	case nbcontractv1.NetworkPolicy_NPO_AZURE:
 		return "azure"
-	case nbcontractv1.NetworkPolicyType_NETWORK_POLICY_TYPE_CALICO:
+	case nbcontractv1.NetworkPolicy_NPO_CALICO:
 		return "calico"
```

Comment:
a note for later, but maybe we should moving all of these consts into a consts.go file.

Reply:
Agreed. Let me do it in the next PR.